src/Sulu/Bundle/PageBundle/Content/Types/PageSelection.php

```diff
@@ -31,53 +31,20 @@
 class PageSelection extends ComplexContentType implements ContentTypeExportInterface, PreResolvableContentTypeInterface
 {
     /**
-     * @var ContentQueryExecutorInterface
+     * @param bool $showDrafts
+     * @param mixed[] $permissions
```

Comment:
is it possible to specify this array more?

Reply:
After the Constructor Property Promotion is in every Bundle I'll clean up the base line. :)